src/main/java/util/FormplayerHereFunctionHandler.java

```diff
@@ -0,0 +1,35 @@
+package util;
+
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.condition.HereFunctionHandler;
+import org.javarosa.core.model.condition.HereFunctionHandlerListener;
+import org.javarosa.core.model.data.GeoPointData;
+import org.javarosa.xpath.XPathArityException;
+
+/**
+ * Created by amstone326 on 12/6/17.
+ */
+public class FormplayerHereFunctionHandler extends HereFunctionHandler {
+
+    private final String browserLocation;
+
+    public FormplayerHereFunctionHandler(HereFunctionHandlerListener listener, String browserLocation) {
+        super();
+        registerListener(listener);
+        this.browserLocation = browserLocation;
+    }
+
+    @Override
+    public Object eval(Object[] args, EvaluationContext ec) throws XPathArityException {
+        alertOnEval();
+        if (browserLocation == null) {
+            return "";
+        }
+        String[] locationData = browserLocation.split(",");
+        double lat = Double.parseDouble(locationData[0]);
```

Comment:
Seems like this could throw some `NumberFormatException`s or `ArrayIndexOutOfBoundsException`s